thirdparty/README.md

```diff
@@ -454,6 +454,24 @@ Files extracted from upstream source:
 - License.txt
 
 
+## rvo2
+
+- Upstream: http://gamma.cs.unc.edu/RVO2/
+- version: 2.0.2
+- License: Apache 2.0
```

Comment:
Third party module licensing info should be added to [COPYRIGHT.txt](https://github.com/godotengine/godot/blob/master/COPYRIGHT.txt) as well.